flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -400,6 +400,24 @@ void createIcebergTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
     }
   }
 
+  // skip the AbstractConstraint comparison
+  public boolean schemaCompare(org.apache.flink.table.api.Schema rawSchema,
+                               org.apache.flink.table.api.Schema newSchema) {
```

Comment:
Since this returns a boolean, can you use `equals` in the name rather than `compare`, which would return an int (like `Long.compare`)?